### DISCUSSION TITLE
Background color noise

### DISCUSSION THREAD

**Discussion by Aurelio-AVB** [martinet101/ElevenClock#957] [Q&A]
Hi, 

For some reason, the adaptive background color has some noise, which is more noticeable in light colors. If I define a background color it works as intended without any noise. is there any setting that may be causing this?
<img width="93" alt="image" src="https://user-images.githubusercontent.com/61214409/192931068-40ca1c71-4c43-466f-ab78-3e903ad066b0.png">

**Comment by marticliment** [comment]
This noise was supposed to mimic the Windows 11 taskbar acryylic texture, but it does not look like it is working. 

Do you have any kind of taskbar modification tool running?

**Reply by Aurelio-AVB** [reply]
I have not, I am running Win11 with transparency which also affects the taskbar, I don't know if it could be conflicting with Elevenclock. I have it configured to always show, so it is trying to adapt to the current window color, not necessarily to the taskbar.
[ElevenClock_log.txt](https://github.com/martinet101/ElevenClock/files/9679676/ElevenClock_log.txt)

I attached the register log I don't know if it would be useful.

**Reply by marticliment** [reply]
Then that must be fixed. I'll convert this to an issue

**Reply by marticliment** [reply]
This issue should be fixed and on the next version you'll be able to disable the blurry texture from the settings window, under appearance settings
